app/frontend/src/components/forms/submission/StatusPanel.vue

```diff
@@ -27,20 +27,76 @@
               @change="statusFields = true"
             />
 
+            <div v-show="statusFields">
+              <div v-if="showRevising">
+                <label>Submitter's Email</label>
+                <v-autocomplete
+                  v-model="assignee"
+                  clearable
+                  dense
+                  :filter="autoCompleteFilter"
+                  :items="formReviewers"
```

Comment:
i wonder if it's better to not populate this field with the form reviewers .. leaving it blank for now. or putting in a dummy <form submitter here> placeholder

Reply:
also maybe it should be a straight v-text field rather than a drop down, to indicate that it will only have one submitter for the first version of this (?)